shell/common/rasterizer.h

```diff
@@ -48,6 +49,54 @@ class AiksContext;
 
 namespace flutter {
 
+// The result status of Rasterizer::Draw. This is only used for unit tests.
+enum class DrawStatus {
+  // The drawing was done without any specified status.
+  kDone,
+  // Failed to rasterize the frame because the Rasterizer is not set up.
+  kNotSetUp,
+  // Nothing was done, because the call was not on the raster thread. Yielded to
+  // let this frame be serviced on the right thread.
+  kYielded,
+  // Nothing was done, because pipeline was empty.
+  kPipelineEmpty,
+  // Nothing was done, because GPU was unavailable.
```

Comment:
```suggestion
  // The drawing was done without any specified status.
  kDone,
  // Failed to rasterize the frame because the Rasterizer is not set up.
  kNotSetUp,
  // Nothing was done because the call was not on the raster thread. Yielded to
  // let this frame be serviced on the right thread.
  kYielded,
  // Nothing was done because the pipeline was empty.
  kPipelineEmpty,
  // Nothing was done because the GPU was unavailable.
```